core-bundle/src/Controller/FrontendModule/RootPageDependentModulesController.php

```diff
@@ -37,8 +37,15 @@ public function __invoke(Request $request, ModuleModel $model, string $section,
             return new Response('');
         }
 
+        $moduleModel = $this->container->get('contao.framework')->getAdapter(ModuleModel::class);
+        $module = $moduleModel->findByPk($modules[$pageModel->rootId]);
+
+        $cssID = StringUtil::deserialize($module->cssID, true);
+        $cssID[1] = trim(sprintf('%s %s', $cssID[1], implode(' ', $model->classes)));
```

Comment:
Possibly undefined array key.

```suggestion
        $cssID[1] = trim(sprintf('%s %s', $cssID[1] ?? '', implode(' ', $model->classes)));
```

Reply:
Fixed in 8d4482a4a5.